iroha_client/tests/unstable_network.rs

```diff
@@ -10,6 +14,18 @@ use tokio::runtime::Runtime;
 
 const MAXIMUM_TRANSACTIONS_IN_BLOCK: u32 = 1;
 
+fn install_panic_hook() -> Result<(), Report> {
+    static INSTALLED: AtomicBool = AtomicBool::new(false);
+    if INSTALLED
+        .compare_exchange(false, true, Ordering::Acquire, Ordering::Relaxed)
+        .is_ok()
+    {
+        color_eyre::install()
+    } else {
+        Ok(())
+    }
+}
+
```

Comment:
I guess you can just call `color_eyre` discarding error, instead of using this function

Reply:
As I understand color_eyre panics, so we need to check if it was installed

Reply:
Correct. It calls set_theme and uses expect.

Reply:
It panics only if it was already called before

Reply:
For atomics case, i guess it needs `AcquireRelease` semantics

Reply:
Better to use `SeqCst` in all test cases though. It never breaks and there is no need for super performance in tests

Reply:
I need to move this function to some place because it's already needed in sumeragi_with_mock.rs
Do you think iroha_logger is good place?